### PR TITLE
Update main menu display

### PR DESCRIPTION
**Pull request by xawen**
- Remove static "Menu Settings" header
- Display 3 items on menu instead of 2
- Active menu item is always in the middle, arrow doesn't move

### CODE REVIEW

File: app/menu.c [hunk 1]
```diff
@@ -131,6 +131,15 @@ uint8_t gSettingsCount = sizeof(Menu) / sizeof(Menu[0]);
...
 static void DrawMenu(uint8_t Index)
 {
+
+	UI_DrawString(24, 72, Menu[(gMenuIndex + gSettingsCount - 1) % gSettingsCount], 14);
+	if (Index==0)
+	{
+		Int2Ascii(gSettingsCount, 2);
+	} else {
+		Int2Ascii((gMenuIndex + gSettingsCount) % gSettingsCount,2);
+	}
```
Comment (reppad) [inline app/menu.c]:
```suggestion
	UI_DrawString(24, 72, Menu[(Index + gSettingsCount - 1) % gSettingsCount], 14);
	Int2Ascii(((Index + gSettingsCount - 1) % gSettingsCount) + 1, 2);
```

**Comment by reppad** [comment]
Clearly better than the basic menu display!

**Comment by reppad** [comment]
When you access a menu by typing its number directly on the keyboard, the display goes over the menu.

**Comment by xawen** [comment]
> When you access a menu by typing its number directly on the keyboard, the display goes over the menu.

I couldn't decide where to put that text when numbers are entered.  I moved it to the blank space off to the left, that seem better.

Merged your improved code.  I tried about 10 different versions of that, and couldn't come up with the right one.  Nice!

**Comment by reppad** [comment]
I tried this, what do you think?

**Comment by xawen** [comment]
> I tried this, what do you think?

Yes!  That's the best one.

**Pull request merged**